spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -592,18 +598,26 @@ class CometSparkSessionExtensions
     }
   }
 
-  // CometExec already wraps a `ColumnarToRowExec` for row-based operators. Therefore,
-  // `ColumnarToRowExec` is redundant and can be eliminated.
+  // This rule is responsible for eliminating redundant transitions between row-based and
+  // columnar-based operators for Comet. Currently, two potential redundant transitions are:
+  // 1. ColumnarToRowExec at the end of a Spark operator, which is redundant for Comet operators as
```

Comment:
Not related to this PR. But the first item reads confused to me. I think this only removes `ColumnarToRowExec` on top of `CometCollectLimitExec`, isn't?